blockchain-albatross/src/blockchain.rs

```diff
@@ -101,9 +101,9 @@ pub struct Blockchain {
 }
 
 pub struct BlockchainState {
-    accounts: Accounts,
-    transaction_cache: TransactionCache,
-    pub(crate) reward_registry: SlashRegistry,
+    pub accounts: Accounts,
```

Comment:
Also allows changing the Accounts state.

Reply:
So, am I supposed to read an account balance then?